src/features/wallet/TwoKeyContractModal.tsx

```diff
@@ -48,20 +97,12 @@ const _TwoKeyContractModal: ForwardRefRenderFunction<
                   <Disconnected tw="flex-grow">Not Connected</Disconnected>
                   <div
                     onClick={() => {
-                      if (votingAddress && network && signer) {
-                        createDesignatedVotingContract(
-                          votingAddress,
-                          signer,
-                          network
-                        ).then((res) => {
-                          console.log("Success dvc?", res);
-                        });
-                      }
+                      closeForm();
```

Comment:
small nit but why not just pass this function directly into the handler `onClick={closeForm}`

Reply:
For which onClick handler?